src/plugins/plot/barGraph/BarGraphConstants.js

```diff
@@ -0,0 +1,13 @@
+export const CONFIGURATION_CHANGED = 'configurationChanged';
+export const HOVER_VALUES_CHANGED = 'hoverValuesChanged';
+export const HOVER_VALUES_CLEARED = 'hoverValuesCleared';
+export const LEGEND_EXPANDED = 'plot-legends-expanded';
```

Comment:
should this be camel case too for consistency?

Reply:
Those were all not being used. Removed them.